payments/mobile-pay/other-features.md

```diff
@@ -16,7 +16,699 @@ sidebar:
+
+### Create authorization transaction
+
+The authorization transaction is initiated by redirecting the end-user/consumer
+to the hyperlink returned in the `redirect-authorization` request.
+
+## Captures
+
+The `captures` resource lists the capture transactions (one or more) on a
+specific payment.
+
+{% include transaction-response.md payment-instrument="mobile-pay"
+    transaction="capture" %}
+
+### Create capture transaction
+
+A `capture` transaction - to withdraw money from the payer's mobilepay - can be
+created after a completed authorization by performing the `create-capture`
+operation.
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/mobilepay/payments/{{ page.paymentId }}/captures HTTP/1.1
+Host: {{ page.apiUrl }}
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "transaction": {
+        "amount": 1000,
+        "vatAmount": 250,
+        "payeeReference": 1234,
+        "description" : "description for transaction"
+    }
+}
+```
+
+{:.table .table-striped}
+| Required | Property                 | Type         | Description                                                                                                               |
+| :------- | :----------------------- | :----------- | :------------------------------------------------------------------------------------------------------------------------ |
+| ✔︎︎︎︎︎   | `transaction`            | `object`     | The currenct capture object.                                                                                              |
+| ✔︎︎︎︎︎   | └➔&nbsp;`amount`         | `integer`    | Amount Entered in the lowest momentary units of the selected currency. E.g. `10000` = `100.00 DKK`, `5000` = `50.00 DKK`. |
+| ✔︎︎︎︎︎   | └➔&nbsp;`vatAmount`      | `integer`    | Amount Entered in the lowest momentary units of the selected currency. E.g. `10000` = `100.00 DKK`, `5000` = `50.00 DKK`. |
+| ✔︎︎︎︎︎   | └➔&nbsp;`description`    | `string`     | A textual description of the capture transaction.                                                                         |
+| ✔︎︎︎︎︎   | └➔&nbsp;`payeeReference` | `string(50)` | A unique reference for the capture transaction. See [`payeeReference`][payeeReference] for details.                       |
+
+The `capture` resource contains information about the capture transaction made
+against a mobilepay payment.
+You can return a specific capture transaction by adding the transaction id to
+the `GET` request.
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "payment": "/psp/mobilepay/payments/{{ page.paymentId }}",
+    "captures": {
+        "id": "/psp/mobilepay/payments/{{ page.paymentId }}/captures",
+        "captureList": [
+            {
+                "id": "/psp/mobilepay/payments/{{ page.paymentId }}/captures/{{ page.transactionId }}",
```

Comment:
Does the `id` of `captureList` really contain a `transactionId`? Shouldn't it just end with `/captures`?

Reply:
Good question, section is replaced by new include.